test/acceptance/framework/config/config.go

```diff
@@ -62,9 +65,9 @@ func (t *TestConfig) HelmValuesFromConfig() (map[string]string, error) {
 		setIfNotEmpty(helmValues, "global.image", entImage)
 	}
 
-	if t.EnterpriseLicenseSecretName != "" && t.EnterpriseLicenseSecretKey != "" {
-		setIfNotEmpty(helmValues, "server.enterpriseLicense.secretName", t.EnterpriseLicenseSecretName)
-		setIfNotEmpty(helmValues, "server.enterpriseLicense.secretKey", t.EnterpriseLicenseSecretKey)
+	if t.EnterpriseLicense != "" {
```

Comment:
Meta comment, no changes needed: I feel like since `EnterpriseLicense` && `EnableEnterprise` are dependent on eachother at this point it'd be great to consolidate them?

Reply:
Oh actually I see that you can set `EnableEnterprise` w/o setting `EnterpriseLicense` and then you pick from the ENV.. nevermind!